test/chart/BarChart.spec.tsx

```diff
@@ -3591,8 +3591,8 @@ describe('<BarChart />', () => {
       fireEvent.mouseOut(barCharts[0]);
       const tooltips3 = container.querySelectorAll('.recharts-tooltip-wrapper');
       // TODO: remove these waits. Right now with both Context and Redux there are too many re-renders.
```

Comment:
Looks like this test is timing out. Maybe we can check to see if we can just remove these now